api/api.go

```diff
@@ -254,6 +254,11 @@ type QueryMeta struct {
 	// CacheAge is set if request was ?cached and indicates how stale the cached
 	// response is.
 	CacheAge time.Duration
+
+	// DefaultACLPolicy is used to control the ACL interaction when there is no
+	// defined policy. This can be "allow" which means ACLs are used to
+	// black-list, or "deny" which means ACLs are white-lists.
```

Comment:
```suggestion
	// deny-list, or "deny" which means ACLs are allow-lists.
```

Reply:
I actually lifted this comment pretty much as-is from `agent/config/runtime.go` and `agent/consul/config.go` so i'll fix them all.